lib/glimesh/community_team.ex

```diff
@@ -0,0 +1,124 @@
+defmodule Glimesh.CommunityTeam do
+  @moduledoc """
+  The Community Team context :)
+  """
+  import Ecto.Query, warn: false
+  alias Glimesh.CommunityTeam.AuditLog
+  alias Glimesh.Repo
+
+  def access_level_to_title(level) do
+    case get_global_access_level(level) do
+      5 -> "Admin"
+      4 -> "Manager"
+      3 -> "Team Lead"
+      2 -> "Team Member"
+      1 -> "Trial Member"
+      _ -> "None"
+    end
+  end
+
+  def can_edit_user(user) do
+    if get_global_access_level(user.gct_level) >= 3, do: true, else: false
```

Comment:
Confused about these functions as well, shouldn't we make the permissions individual knobs? `can_edit_user`, `can_ban_user`, etc?

Reply:
That is also possibly but I'm more afraid of the clutter that would cause. Having it set to a number level allows for more permissions to be added to someone without having to go through and edit the entire team.